data/remnant/remnant ships.txt

```diff
@@ -893,47 +893,32 @@ ship "Merganser"
 		licenses
 			Remnant
 		"cost" 6830000
-		"shields" 3000
-		"hull" 5000
+		"shields" 3500
+		"hull" 3000
 		"required crew" 1
 		"bunks" 3
 		"cargo space" 11
 		"fuel capacity" 600
-		"ramscoop" 1.75
-		"mass" 30
+		"ramscoop" 1.25
```

Comment:
If we're going that route, there have been tens of thousands of changes that haven't been explained on github over the years. "Because it changed" is not a justification.

Reply:
The justification given was that it was an adjustment in response to the changes to the ramscoop scaling in the Ember Waste (an addend of -.02, and a multiplier of 1.5). These changes _increased_ the average gain of fuel from ramscooping for the Merganser, and for most other ships, with the exception of some of the smallest ships with only 0.5 ramscoop stat. A change that increases the fuel gained by ramscooping does not, in my mind, justify an increase of the ramscoop value "in response". So, that's what I meant when I told Azure.

Leaving that aside, that PR (8201) was specifically merged with the comments "Anything merged is still susceptible to change." and "There's nothing stopping us if we wish to change it in the future anyway". And, subsequent changes are now being proposed.

Reply:
I agree with you. This isn't a Merganser issue. All remnant ships had ramscoops buffed, not just the Merganser. I think that should be addressed separately.